frontend/src/modules/policy/form.jsx

```diff
@@ -173,18 +173,12 @@ const AddPolicyForm = withRouter(
     highlight,
     setHighlight,
     setDisabledBtn,
+    isLoaded,
     history,
     match: { params },
   }) => {
-    const {
-      countries,
-      organisations,
-      tags,
-      loading,
-      formStep,
-      formEdit,
-    } = UIStore.currentState;
-
+    const globalState = UIStore.useState();
```

Comment:
same here ... https://lostpebble.github.io/pullstate/docs/use-store-state-hook#use-a-sub-state-of-a-store